h/models/organization.py

```diff
@@ -47,16 +47,18 @@ def validate_name(self, key, name):
 
     @sa.orm.validates('logo')
     def validate_logo(self, key, logo):
-        if not (len(logo) <= ORGANIZATION_LOGO_MAX_CHARS):
-            raise ValueError(
-                'logo must be less than {max} characters long'
-                .format(max=ORGANIZATION_NAME_MAX_CHARS))
-        try:
-            root = ElementTree.fromstring(logo)
-        except ElementTree.ParseError:
-            raise ValueError('logo is not a valid SVG (could not parse XML)')
-        if root.tag != 'svg':
-            raise ValueError('logo is not a valid SVG (does not start with an <svg> tag')
+        # If the logo is not None then it must be an svg format.
+        if logo:
```

Comment:
I personally find it easier to read functions written like this:

```py
if some_condition:
  # early exit for alternate path
  return foo

# lots of code for the "happy path"
```

Rather than:

```py
if some_condition:
  # lots of code for the happy path
else:
  # alternate path
```

The former reduces indentation and makes it easier to follow the logic for the `else` path as it keeps the test & consequence together.